File: src/Other/Internals/Search/SearchKeywordAttribute.vb
Namespace OpenSilver.Samples.Showcase.Search
    <AttributeUsage(AttributeTargets.[Class], AllowMultiple:=False)>
    Public Class SearchKeywordsAttribute
        Inherits Attribute

        Public ReadOnly Property Keywords As String()

        Public Sub New(ParamArray keywords As String())
            Me.Keywords = keywords
        End Sub
    End Class
End Namespace

File: src/Other/Internals/Search/SearchKeywordAttribute.fs
namespace OpenSilver.Samples.Showcase.Search

open System

[<AllowNullLiteral>]
[<AttributeUsage(AttributeTargets.Class, AllowMultiple = false)>]
type SearchKeywordsAttribute([<ParamArray>] keywords: string[]) =
    inherit Attribute()

    member _.Keywords = keywords
